doc/maintaining/installing/solr.rst

```diff
@@ -12,7 +12,22 @@ installed, we need to install and configure Solr.
    server, but CKAN doesn't require Jetty - you can deploy Solr to another web
    server, such as Tomcat, if that's convenient on your operating system.
 
-#. Edit the Jetty configuration file (``/etc/default/jetty8`` or
+.. tip::
+
+   Do this step only if your system is using ``jetty9``.
```

Comment:
It's probably best to be explicit and say "Do this step only if you are using Ubuntu 18.04."

Reply:
Since Ubuntu 18.04 for some reason supports only jetty9, yes, this might be the better solution.